src/dstack/_internal/core/backends/gcp/compute.py

```diff
@@ -116,7 +136,68 @@ def create_instance(
             project_id=self.config.project_id,
         )
         disk_size = round(instance_offer.instance.resources.disk.size_mib / 1024)
+        commands = get_shim_commands(authorized_keys=authorized_keys)
+        startup_script = " ".join([" && ".join(commands)])
+        startup_script = "#! /bin/bash\n" + startup_script
+        instance_id = f"tpu-{instance_config.instance_name}"
+        if "tpu" in instance_offer.instance.resources.gpus[0].name:
+            tpu_node = gcp_resources.create_tpu_node_struct(
+                instance_name=instance_offer.instance.name,
+                startup_script=startup_script,
+                authorized_keys=authorized_keys,
+                spot=instance_offer.instance.resources.spot,
+                project_id=self.config.project_id,
+                zone=instance_offer.region,
```

Comment:
For regular VMs, we try provisioning in all zones in the region. Why trying only one zone for TPUs?

Reply:
Same as above. I will fix it.